dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.1/bnd.bnd

```diff
@@ -21,11 +21,24 @@ src: \
   
 fat.project: true
 
-#tested.features: jsp-2.3, jdbc-4.1, el-3.0
-#1/29/2021 update
-tested.features: jsp-2.3, jdbc-4.1, el-3.0, appsecurity-3.0, servlet-4.0, jaxb-2.3, cdi-2.0, jaxws-2.3, federatedregistry-1.0, ldapregistry-3.0
-
-
+tested.features:\
+  jdbc-4.1, \
+  jaxws-2.3, \
+  xmlWS-3.0, \
+  jaxb-2.3, \
+  xmlBinding-3.0, \
+  jsp-2.3, \
+  pages-3.0, \
+  servlet-4.0, \
+  servlet-5.0, \
+  appSecurity-3.0, \
+  appSecurity-4.0, \
+  cdi-2.0, \
+  cdi-3.0, \
+  el-3.0, \
+  expressionlanguage-4.0
```

Comment:
was federatedregistry-1.0 and ldapregistry-3.0 purposefully not included in this new list?

Reply:
yes. they are not used in these FAT (ldap user registry is not used but basicRegistry). They were used at one point during FAT migration and after learning more about the testing, it's time for minor cleanup.